### DISCUSSION TITLE
Cannot use with AWS EC2

### DISCUSSION THREAD

**Discussion by abc25339** [angristan/openvpn-install#1404] [Help]
I succeed in a Vultr server. But I cannot connect from OpenVPN Client if I install the VPN server in AWS EC2

**Comment by MrFranke** [comment]
I had similar issue, but my problem was be is wrong client version. Check your client version it's should be 2.x version, not 3.x
Try to use one of clients from FAQ: https://github.com/angristan/openvpn-install#faq

**Comment by antoweb** [comment]
I have installed in AWS ec2 debian buster without problem. Remember to disable source/dest check in nic on aws ec2 server. And add route in subnet routing table that redirect the vpn subnet on the instance where is isntalled openvpnserver

**Comment by chrispro-21** [comment]
I've been running it in a ubuntu EC2 instance for a while now with no issue. odd

**Comment by angelosnm** [comment]
> I've been running it in a ubuntu EC2 instance for a while now with no issue. odd

me too.
No issue running the service on AWS instance

**Comment by Ppang0405** [comment]
odd. I got issue with aws ec2 and lightsail, try with openvpn-install and https://github.com/angristan/wireguard-install
Connection always timeout, in my iphone and mac.

Vultr server is just fine.

**Comment by genieai-vikas** [comment]
Even I am facing the issue on AWS EC2

```
2021-11-26 11:34:06.142627 TLS Error: TLS key negotiation failed to occur within 60 seconds (check your network connectivity)
2021-11-26 11:34:06.142686 TLS Error: TLS handshake failed
2021-11-26 11:34:06.142848 SIGUSR1[soft,tls-error] received, process restarting
2021-11-26 11:34:06.142920 MANAGEMENT: >STATE:1637926446,RECONNECTING,tls-error,,,,,
2021-11-26 11:34:06.158755 MANAGEMENT: CMD 'hold release'
2021-11-26 11:34:06.158804 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
```

**Comment by MahaGamal** [comment]
I had the same issue and i solved it by using TCP instead of UPD

**Comment by marcustut** [comment]
> I had the same issue and i solved it by using TCP instead of UPD

I think you only had to allow UDP port on 1194 in the security group or whichever port you chose during setup for it to work
